src/Fabulous.AST.Tests/TypeDefinitions/Class.fs

```diff
@@ -190,7 +200,7 @@ type Person (name: string) =
         let param =
             SimplePatNode(None, false, SingleTextNode("name", Range.Zero), Some(Type.FromString("string")), Range.Zero)
 
-        AnonymousModule() { (Class("Person", [ param ]) { EscapeHatch(memberNode) }).isStruct() }
+        AnonymousModule() { (Class("Person") { EscapeHatch(memberNode) }).isStruct().parameters([ param ]) }
```

Comment:
Nit, but I find `.parameters` here confusing and vague.
You mean the `implicitConstructorParameters` here.